CONTRIBUTING.md

```diff
@@ -0,0 +1,287 @@
+# Contributing
+
+Contributions are welcome from anyone, regardless of whether you are just starting your Python journey, a web developer or a seasoned Execution Layer developer!
+
+We appreciate your help and appreciate your contributions!
+
+## Code of Conduct
+
+All contributors are expected to be excellent to each other; other behavior is not tolerated.
+
+## Communication
+
+We encourage questions and discussions about the project. If you need help with the codebase or have questions about implementation details, please don't hesitate to reach out in the `#el-testing` channel in the [Ethereum R&D Discord Server](https://discord.com/invite/qGpsxSA).
+
+For detailed information on how to get help, please see the [Getting Help](https://eest.ethereum.org/main/getting_started/getting_help) page in our documentation, which includes communication channels and contact information for project maintainers.
+
+## Contributions We Welcome
+
+As mentioned in the README's [Contributing section](https://github.com/ethereum/execution-spec-tests#contributing), we welcome earnest contributions that have reasonable substance or resolve existing repository issues.
+
+## Contributions We Don't Accept
+
+We do not accept:
+
+- Contributions that only fix spelling or grammatical errors in documentation, code, or elsewhere.
+- Drive-by or vibe code contributions without proper engagement or context.
+- Pull requests from airdrop farmers.
+
+I.e., pull requests should have reasonable substance and context.
+
+## Reporting Bugs
+
+We use GitHub Issues to track bugs. To report a bug, please follow these guidelines:
+
+### Before Reporting
+
+1. **Check existing issues**: Search [open issues](https://github.com/ethereum/execution-spec-tests/issues) to see if your problem has already been reported.
+2. **Try to reproduce**: Confirm you can reproduce the issue consistently.
+3. **Consider security implications**: For security vulnerabilities, please do NOT create a public issue (or PR). Instead, refer to our [Security Policy](SECURITY.md) for responsible disclosure guidelines.
+
+### Creating a Bug Report
+
+When creating a new issue:
+
+1. **Use a clear, descriptive title** that identifies the problem, (see [Commit Messages, Issues and PR Titles](#commit-messages-issue-and-pr-titles)).
+2. **Provide detailed reproduction steps**:
+    - Include the exact commands you ran.
+    - Share relevant console output.
+    - Specify your environment (OS, Python version, if relevant, `uv` version from `uv --version`).
```

Comment:
I'm wondering if at some point we should add a `eest info` cli that may prints this out:
```
EEST v4.5.0 (7abf18e)

Python: 3.12.3
uv: 0.6.14
OS : Windows 10
```
This could be helpful for quick debugging, say if `uv` for example is out of date. We could also create a bug report template that asks to provide the output of this command.

Reply:
https://github.com/ethereum/execution-spec-tests/issues/1619

Reply:
#1621 adds the command, if you want update the docs.